lingua_franca/lang/parse_common.py

```diff
@@ -201,7 +201,7 @@ def match_yes_or_no(text, lang):
         words = json.load(f)
         words = {k: [_.lower() for _ in v] for k, v in words.items()}
     # after encoding information is lost
-    if lang == 'uk-uk':
+    if lang == 'uk-ua':
```

Comment:
Should this be extended to other non-ascii languages? (`if lang in ('uk-ua',)`)

Reply:
That seems like it should be a utility flag and getter unto itself, to avoid hardcoding it wherever it comes up

Reply:
Any good ideas how to implement that here, or should this be a TODO with an issue to discuss more?